traits-stubs/MANIFEST.in

```diff
@@ -0,0 +1,4 @@
+include LICENSE.txt
+include MANIFEST.in
+include README.rst
+recursive-include traits *.pyi
```

Comment:
Is `traits` right here? Should it be `traits-stubs`?

Reply:
D'oh! Yep. Fixing.

Reply:
FWIW, I previously checked that all the `.pyi` files _are_ contained in the sdist, so apparently this isn't needed at all...